10.11/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM quay.io/centos7/s2i-core-centos7
+
+# MariaDB image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql/data - Datastore for MariaDB
+# Environment:
+#  * $MYSQL_USER - Database user name
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+#  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
+#  * $MYSQL_CHARSET (Optional) - Default character set
+#  * $MYSQL_COLLATION (Optional) - Default collation
+
+ENV MYSQL_VERSION=10.5 \
```

Comment:
Why should we have version 10.5 in 10.11 directory? Even Dockerfile for RHEL7 doesn't make sense to me if we do not support 10.11 in RHEL7 images

Reply:
Fixed by https://github.com/sclorg/mariadb-container/pull/223/commits/746dc057b928981308fd521d70ddf3ae7ece29f1